config/auth.go

```diff
@@ -43,7 +43,7 @@ func Auth(ctx context.Context) (*scalingo.User, string, error) {
 			if err == nil {
 				break
 			} else if scalingoerrors.RootCause(err) == io.EOF {
```

Comment:
issue(non-blocking): Can you change the deprecated function `.RootCause()` to the function `.Is()`